server/schedule/operator/step.go

```diff
@@ -38,6 +38,14 @@ const (
 	// DefaultFastExecutorRate is the slow rate of the operator executor.
 	// default:  0.1 s/Mb
 	DefaultFastExecutorRate = 0.1
```

Comment:
Should change DefaultFastExecutorRate？because SlowStepWaitTime/FastStepWaitTime is changed to 10 from 60

Reply:
fixed to 0.6